pacemaker.spec.in

```diff
@@ -1,77 +1,114 @@
-%global gname haclient
+# Define globals to control package behavior (configure these as desired)
+
+## User and group to use for nonprivileged services
 %global uname hacluster
+%global gname haclient
+
+## Where to install Pacemaker documentation
 %global pcmk_docdir %{_docdir}/%{name}
 
-%global specversion 1
+## GitHub entity that distributes source (for ease of using a fork)
+%global github_owner ClusterLabs
+
+## Upstream pacemaker version, and its package version
```

Comment:
From my experience, it would deserve, akin to previous explanation, `(leveraged in automation)` (meaning "make (s)rpm", indeed).